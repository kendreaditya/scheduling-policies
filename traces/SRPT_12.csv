1,0,200
2,0,0
3,100,100
4,200,101
5,201,100
6,400,300
7,400,0
8,400,0
9,400,0
10,600,5
11,600,4
12,600,3
13,1000,1500
14,1000,0
15,1000,0
16,1000,0
17,1000,0
18,1000,0
19,1000,0
20,1000,0
21,2000,500
22,3000,1500
23,4000,500
24,5000,1500
25,6000,500
26,7000,1500
27,8000,500
28,9000,1500
29,10000,500
30,11000,0
