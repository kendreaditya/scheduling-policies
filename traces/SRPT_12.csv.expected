1,200
2,0
3,300
4,501
5,400
6,813
7,400
8,400
9,400
10,612
11,607
12,603
13,2500
14,1000
15,1000
16,1000
17,1000
18,1000
19,1000
20,1000
21,3000
22,4500
23,5000
24,6500
25,7000
26,8500
27,9000
28,10500
29,11000
30,11000
